teams/helpers.py

```diff
@@ -0,0 +1,166 @@
+from copy import deepcopy
+from itertools import combinations
+import json
+import math
+
+LMS_LEVELS = {
+    "programme_preliminaries": 1,
+    "programming_paradigms": 1,
+    "html_fundamentals": 1,
+    "css_fundamentals": 1,
+    "user_centric_frontend_development": 1,
+    "javascript_fundamentals": 2,
+    "interactive_frontend_development": 2,
+    "python_fundamentals": 3,
+    "practical_python": 3,
+    "data_centric_development": 3,
+    "full_stack_frameworks with django": 4,
```

Comment:
Missing `_` underscores, uses `space` instead.

Also, do we want to go with the new LMS module names? For example: `css_essentials` instead of `css_fundamentals`, since all/new students are migrated onto the new LMS?

Reply:
Added missing `_`s. Changed from `fundamentals` to `essentials` and added `backend_development`